### DISCUSSION TITLE
Whether a new compilation mode can be added

### DISCUSSION THREAD

**Discussion by jipika** [quasarframework/quasar#13572] [Ideas / Proposals]
Hope to join China's small program can also be adapted to compile , Applets are in great demand

**Comment by smolinari** [comment]
@jipika - Can you expand on this? It makes little sense, unfortunately. 

Scott

**Reply by jipika** [reply]
Such as wechat mini programs   https://developers.weixin.qq.com/miniprogram/dev/framework/ 
It's similar to PWA, but it runs inside apps like wechat or Alipay

**Reply by smolinari** [reply]
Ah. Ok. Currently we don't have anything like this on the road map. So, probably won't be happening anytime soon, if at all. We are always happy to get PRs for such features though, if you wish to tackle it. 

Scott
